### PR TITLE
Upload task return incorrect size

### PR DESCRIPTION
**Pull request by iNikitaGricenko**
### What changes are being made and why?

- Was changed code that setting size of files #110, because for Google Drive files such as doc, spreadsheet, csv, etc - file.getSize() will return null, as they don't consumer any space in drive
- Versioning issue, as far, as i know, seems like occurs only for text/csv contentType on Google Drive side
- Changed tests
---

### How the changes have been QAed?

```yaml
tasks:
  - id: "upload"
    type: "io.kestra.plugin.googleworkspace.drive.Upload"
    from: "{{ inputs.file }}"
    parents:
     - "parent-uuid-123x7kd"
    name: "order.csv"
    contentType: "text/csv"
    mimeType: "application/vnd.google-apps.spreadsheet"
    serviceAccount: "{{ secret('GCP_SERVICE_ACCOUNT_JSON') }}"
```

### CODE REVIEW

File: src/main/java/io/kestra/plugin/googleworkspace/drive/Export.java [hunk 1]
```diff
@@ -73,6 +73,10 @@ public Output run(RunContext runContext) throws Exception {
 
             runContext.metric(Counter.of("size", file.size()));
 
+            // For Google apps files such as doc, spreadsheet, csv - file.getSize() will return: null, 1 or 1024 (Which is incorrect value)
+            // Hardcoding file size from tempFile size in bytes that have been exported/created
+            file.setSize(tempFile.length());
+
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/googleworkspace/drive/Export.java:79]: Can you move that before the metric just above so the metric correctly use the right size?

**Review by loicmathieu — CHANGES_REQUESTED** [review]

File: src/main/java/io/kestra/plugin/googleworkspace/drive/Export.java (hunk 1 above)
Comment (iNikitaGricenko) [reply src/main/java/io/kestra/plugin/googleworkspace/drive/Export.java:79]: Metric uses different size anyway (```size()``` return size of map as File extends of AbstractMap. ```getSize()``` returns size of file). I did want to change that, but wasn't sure if that was intended)

Comment (loicmathieu) [reply src/main/java/io/kestra/plugin/googleworkspace/drive/Export.java:79]: OK, let's keep this one as is then.

**Review by loicmathieu — APPROVED** [review]
LGTM

**Pull request merged**